bundles/content/reference-store.rst

```diff
@@ -0,0 +1,52 @@
+Reference Store
+===============
+
+The reference-store is a service which collects ids of entities/documents
+which are used to render a page. This ids will be used for example in the
+caching component :doc:`../http_cache`.
+
+Architecture
+------------
+
+This component is build with the "composition pattern" in mind. Each
```

Comment:
It's not a composition pattern, since the `ReferenceStorePool` and the `ReferenceStore` are not sharing any base class or interface, and the structure which is built is not really a tree.